test/service/test_request_validation.py

```diff
@@ -328,3 +328,12 @@ def test_size(self):
             with self.subTest(size=size):
                 url.args.set('size', size)
                 test(url, arg)
+
+    def test_order(self):
+        supported_orders = {'asc', 'desc'}
+        url = self.base_url.set(path='/index/projects')
+        for order in ['a', 'ascx', 'foo', 1]:
+            with self.subTest(order=order):
+                url.args.set('order', order)
+                self.assertBadRequest(url, f'Unknown order `{order}`. '
+                                           f'Must be one of {supported_orders}')
```

Comment:
Inline `supported_orders` so there is an example of what this part of the message ultimately looks like.